src/main/java/com/sprint/mission/discodeit/entity/base/BaseEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.sprint.mission.discodeit.entity.base;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.Id;
+import jakarta.persistence.Inheritance;
+import jakarta.persistence.InheritanceType;
+import jakarta.persistence.MappedSuperclass;
+import java.time.Instant;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@Getter
+@Setter
+@EntityListeners(AuditingEntityListener.class)
+public abstract class BaseEntity {
+  @Id
+  private UUID id;
+  @CreatedDate
+  private Instant createdAt;
+
+  public BaseEntity(){
```

Comment:
접근제어자를 protected로 바꿔도 될 것 같아요.  
```suggestion
  protected BaseEntity(){
```